packages/base/src/widget.ts

```diff
@@ -886,6 +886,10 @@ export class DOMWidgetView extends WidgetView {
     this.updateTooltip();
   }
 
+  get pWidget(): Widget {
```

Comment:
Should there be a docstring to mention that using `pWidget` is deprecated?

Reply:
Done, thanks.